Makefile.am

```diff
@@ -151,7 +151,6 @@ test_run_tests_SOURCES = test/blackhole-server.c \
                          test/test-default-loop-close.c \
                          test/test-delayed-accept.c \
                          test/test-dlerror.c \
-                         test/test-eintr-handling.c \
```

Comment:
you also need to remove it from uv.gyp

Reply:
This needs to be addressed

Reply:
@santigimeno: This commit already removes it from uv.gyp, what else needs to be addressed?

Reply:
@bwijen disregard that comment, I missed the change, sorry.